src/libcrun/utils.h

```diff
@@ -323,7 +323,7 @@ int copy_from_fd_to_fd (int src, int dst, int consume, libcrun_error_t *err);
 
 int run_process (char **args, libcrun_error_t *err);
 
-size_t format_default_id_mapping (char **ret, uid_t container_id, uid_t host_id, int is_uid);
+size_t format_default_id_mapping (char **ret, uid_t container_id, uid_t host_uid, uid_t host_id, int is_uid);
```

Comment:
Shouldn't this be (..., uid_t host_uid, uid_t host_gid,...)

Reply:
`host_id` is either UID or GID, depending on the value of `is_uid`.  We need to differentiate since we use different files for the lookup (`/etc/subuid` or `/etc/subgid`)